### DISCUSSION TITLE
添加评论组件

### DISCUSSION THREAD

**Discussion by AiYuZhen** [dotnetcore/BootstrapBlazor#4825] [Q&A]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your feature request related to a problem? Please describe the problem.

请大佬们增加一个 [Comment](https://antblazor.com/zh-CN/components/comment) 评论组件。

### Describe the solution you'd like

类似于这种实现 https://antblazor.com/zh-CN/components/comment

**Comment by bb-auto** [comment]
@AiYuZhen Thank you for contacting us. We will give feedback later.

**Comment by ArgoZhang** [comment]
@AiYuZhen 网站打不开，尴尬。有同款吗？

**Comment by AiYuZhen** [comment]
通过这个网址可以打开的：[ https://antblazor.com/zh-CN/components/comment]( https://antblazor.com/zh-CN/components/comment)
![image](https://github.com/user-attachments/assets/aba6fff1-43cc-4041-8bc5-d754a32b90cc)

**Comment by densen2014** [comment]
要做论坛就要挂在外网了，你懂的。不审核就发布要连坐

**Comment by AiYuZhen** [comment]
谢谢大佬提醒，不是搞论坛，只公司内部使用的功能加上评论，如：日志填报，领导可以在下面进行点评；工作流程，可以在每一个环节留下评论。

**Comment by ArgoZhang** [comment]
@AiYuZhen 这种东西根本不需要组件啊。就是一个样式问题。网页我这里打不开。确实打不开。他们的一直是 wasm 打不开这个问题好些年了

**Comment by AiYuZhen** [comment]
我试着弄了个半成品，年底了任务重，还没有时间完善
![image](https://github.com/user-attachments/assets/2ca94b2c-89d8-4020-a978-ea3f2328f33e)

**Comment by densen2014** [comment]
如果不是公开的系统，那倒没关系。 不过这东西真心不需要做成组件。

**Comment by AiYuZhen** [comment]
行嘞，我觉得我写这个就能满足我的需求了。
可以把这个需求关闭了。

**Comment by ArgoZhang** [comment]
@AiYuZhen 这个我看了一下。实在是没有做组件的必要，这个布局基本上也是只能做一个最简单的。然后要开放一个 `Template` 因为根据自己的需求大概率要改，比如你是左右，右侧又是上下。其实做 `Template` 的时候基本这个组件就做完了。所以即使我们出一个组件也是一个壳，实话实说就是增加了 html 的负担
